test/suite/test_tiered13.py

```diff
@@ -140,19 +138,3 @@ def test_tiered13(self):
         # Try to import via the file:uri with file metadata.
         self.assertRaisesWithMessage(wiredtiger.WiredTigerError,
             lambda: self.session.create(self.file2uri, import_meta), invalid)
-
-        # Try to import via a renamed object. If we don't send in metadata,
```

Comment:
This test I am unsure about; we are renaming the file (it says object here though) without using the session rename API. However, if we are removing this feature then is this testing redundant?

Reply:
Looking at the code above, I think this rename was done via copying the file. So this isn't a `session.rename` usage, and should probably stay. One way to check would be to make `session.rename` always fail in WiredTiger, and see if this test still passes.

Reply:
Hi @korteland, I agree, I do not think it is using the session rename API (as previously mentioned) - I am more so concerned with whether or not we want to continue testing for a scenario that we aren't going to be using in the future - thanks!

Reply:
Sorry for the delay getting back to you. I think this test should remain. I don't think the rename is the core of the test here - rather, for tiered storage, it appears to be valid to have "table:xyz-0002.wt" present without the metadata knowing about it, and if that happens I can easily see us needing to be able to import it.

I'm not an expert though - I puzzled this out from staring at the code, so if you want to pull in someone that knows more about tiered storage, that would make sense.

Reply:
I am happy to put the test back - most of the test has been disabled and is waiting on another ticket WT-8664 which is unrelated to the rename API - so I think it can be revisited then.

Reply:
Sounds like a plan :-) Thanks for being so patient with this!

Reply:
[365a0c5](https://github.com/wiredtiger/wiredtiger/pull/10282/commits/365a0c5a173cd414c9dfe504c585362aa8464a08)